score/etl/runner.py

```diff
@@ -0,0 +1,10 @@
+dataset_list = ["census_acs", "ejscreen"]  # this list comes from YAMLs
+
+# Run the ETLs for the dataset_list
+pass
+
+# Recalculate score
+pass
+
+# update the front end JSON/CSV of list of data sources
+pass
```

Comment:
Are these todos for later?

Reply:
Yeah -- there's already some more code in that file :)